components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -962,48 +975,55 @@ else if (seriesCount > mosaics && mosaics > 1 && prestitched) {
 
     // find and add attached label/overview images
 
+    boolean readAttachments = INCLUDE_ATTACHMENTS_DEFAULT;
+    if (options instanceof FormatOptions) {
+      readAttachments = ((FormatOptions) options).getBoolean(INCLUDE_ATTACHMENTS_KEY, INCLUDE_ATTACHMENTS_DEFAULT);
+    }
     boolean foundLabel = false;
     boolean foundPreview = false;
-    for (Segment segment : segments) {
-      if (segment instanceof Attachment) {
-        AttachmentEntry entry = ((Attachment) segment).attachment;
-        String name = entry.name.trim();
-
-        if ((name.equals("Label") && !foundLabel) ||
-          (name.equals("SlidePreview") && !foundPreview))
-        {
-          if (!foundLabel) {
-            foundLabel = name.equals("Label");
-          }
-          if (!foundPreview) {
-            foundPreview = name.equals("SlidePreview");
-          }
-          segment.fillInData();
+    if (readAttachments) {
+      for (Segment segment : segments) {
```

Comment:
My 2c on the above:
- no strong argument against not using `classname` in favor of more generic ones, i.e. `zeissczi.xxxz works and also matches the convention in `LociPrefs`
- agreed with all recommendations of Simone, I would vote for lowercase across the board
- with regarding to length vs complexity, this will undoubtedly be a case by case decision. For instance, in this PR `zeissczi.include.attachments` could be simplified/shorterned as `zeissczi.include_attachments` or even `zeissczi.attachments`. On the other hand, if we think there might be similar options in the future, having a generic `zeissczi.include.xxx` makes complete sense
- a final point re maintenance. Like all other properties, these will effectively be public API and we will have to consider how to handle deprecation/upgrade.